lib/src/main/java/com/diffplug/spotless/java/ModuleHelper.java

```diff
@@ -102,7 +102,9 @@ private static void openPackages(Collection<String> packagesToOpen) throws Throw
 		if (modules == null) {
 			return;
 		}
-		final Unsafe unsafe = Unsafe.getUnsafe();
```

Comment:
Otherwise it will cause a failed privilege check:

```java
    @CallerSensitive
    public static Unsafe getUnsafe() {
        Class<?> caller = Reflection.getCallerClass();
        if (!VM.isSystemDomainLoader(caller.getClassLoader()))
            throw new SecurityException("Unsafe");
        return theUnsafe;
    }
```